client/src/models/enums/Gender.ts

```diff
@@ -0,0 +1,6 @@
+enum Gender {
```

Comment:
you're using this strings PersonalInfoTab, so why not import this enum?

Reply:
it is

Reply:
oh nevermind i see what you mean